features/load-rest.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress through endpoints locally
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+	When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
I've filed an issue to include a linter in the future https://github.com/wp-cli/scaffold-package-command/issues/82